src/todos.js

```diff
@@ -0,0 +1,31 @@
+export default class Todos {
+  constructor() {
+    this.list = localStorage.getItem('todos')
+      ? JSON.parse(localStorage.getItem('todos'))
+      : [];
+  }
+
+  addTodo(todo) {
+    this.list.push(todo);
+    localStorage.setItem('todos', JSON.stringify(this.list));
+  }
+
+  removeTodo(todoID) {
+    this.list = this.list.filter((todo) => todo.index !== todoID);
+    this.list.forEach((todo, index) => {
+      todo.index = index;
+    });
+    console.log(this.list);
+    localStorage.setItem('todos', JSON.stringify(this.list));
+  }
```

Comment:
- [ ] Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. currently when I remove one task see what happen
if I have these tasks
![todo](https://user-images.githubusercontent.com/44745855/149162985-1227bcc0-b6aa-4951-961d-5dcbabc10ceb.png)

![td](https://user-images.githubusercontent.com/44745855/149163041-ae39776f-1f37-4f26-a84c-6871bab19f0b.png)

When I remove `cleaning` that means `hiking` will update to the index `1` not `0`
![hike](https://user-images.githubusercontent.com/44745855/149163400-1518231e-ae21-471d-b76b-f67e056dc553.png)